tests/Microsoft.Bot.Builder.ApplicationInsights.Tests/TelemetryWaterfallTests.cs

```diff
@@ -182,17 +204,31 @@ public async Task EnsureCancelDialogCalled()
             var telemetryClient = new Mock<IBotTelemetryClient>();
             var saved_properties = new Dictionary<string, IDictionary<string, string>>();
             var counter = 0;
+
             // Set up the client to save all logged property names and associated properties (in "saved_properties").
             telemetryClient.Setup(c => c.TrackEvent(It.IsAny<string>(), It.IsAny<IDictionary<string, string>>(), It.IsAny<IDictionary<string, double>>()))
                             .Callback<string, IDictionary<string, string>, IDictionary<string, double>>((name, properties, metrics) => saved_properties.Add($"{name}_{counter++}", properties))
                             .Verifiable();
 
-            var waterfallDialog = new MyWaterfallDialog("test", new WaterfallStep[]
+            var steps = new WaterfallStep[]
             {
-                    async (step, cancellationToken) => { await step.Context.SendActivityAsync("step1"); return Dialog.EndOfTurn; },
-                    async (step, cancellationToken) => { await step.Context.SendActivityAsync("step2"); return Dialog.EndOfTurn; },
-                    async (step, cancellationToken) => { await step.CancelAllDialogsAsync(); return Dialog.EndOfTurn; },
-            });
+                    async (step, cancellationToken) =>
+                    {
+                        await step.Context.SendActivityAsync("step1");
+                        return Dialog.EndOfTurn;
+                    },
+                    async (step, cancellationToken) =>
+                    {
+                        await step.Context.SendActivityAsync("step2");
+                        return Dialog.EndOfTurn;
+                    },
+                    async (step, cancellationToken) =>
+                    {
+                        await step.CancelAllDialogsAsync();
+                        return Dialog.EndOfTurn;
+                    },
+            };
+            var waterfallDialog = new MyWaterfallDialog(string.Empty + "test", steps);
```

Comment:
is the string.Empty added for some particular reason?

Reply:
Agreed, the string.Empty is not doing anything. It has been removed.